__tests__/app.test.js

```diff
@@ -29,34 +29,72 @@ describe("GET: /api", () => {
+        expect(typeof body.article.created_at).toBe("string");
+        expect(typeof body.article.votes).toBe("number");
+        expect(body.article.votes).toBe(100);
+        expect(typeof body.article.article_img_url).toBe("string");
+        expect(body.article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
       });
   });
-  test("GET: 404 - returns a message letting the user know the article id does not exist", () => {
+  test("Should return the details for article 2 as that is the endpoint being navigated to", () => {
     return request(app)
-      .get("/api/articles/1000")
-      .expect(404)
+      .get("/api/articles/2")
+      .expect(200)
       .then(({ body }) => {
-        expect(body.msg).toBe("article does not exist");
+        expect(body.article.article_id).toBe(2);
+        expect(typeof body.article.title).toBe("string");
+        expect(body.article.title).toBe("Sony Vaio; or, The Laptop");
+        expect(typeof body.article.topic).toBe("string");
+        expect(body.article.topic).toBe("mitch");
+        expect(typeof body.article.author).toBe("string");
+        expect(body.article.author).toBe("icellusedkars");
+        expect(typeof body.article.body).toBe("string");
+        expect(body.article.body).toBe("Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.");
+        expect(typeof body.article.created_at).toBe("string");
+        expect(typeof body.article.article_img_url).toBe("string");
+        expect(body.article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
       });
   });
+  describe("GET: 404 /api/articles/:article_id", () => {
+    test("Should return a message saying the article id does not exist", () => {
+      return request(app)
+        .get("/api/articles/1000")
```

Comment:
why is 1000 special? You could store it in a const called `NON_EXISTANT_ID` and then it would give the code added meaning